packages/dashboard/src/vaadin-dashboard.js

```diff
@@ -221,7 +221,11 @@ class Dashboard extends DashboardLayoutMixin(ElementMixin(ThemableMixin(PolylitM
 
   /** @protected */
   render() {
-    return html`<div id="grid"><slot></slot></div>`;
+    return html`<div id="grid">
+      <div id="slots" style="display: contents">
```

Comment:
Is this wrapper needed? I'm not sure what its purpose is, as the reordering works correctly without it.

Reply:
Not needed indeed, removed